azure-simple/Packer/ansible-apim/docs/Pattern_1.md

```diff
@@ -0,0 +1,99 @@
+3. [MySQL Connector/J](https://dev.mysql.com/downloads/connector/j/5.1.html)
+
+## Customize the WSO2 Ansible scripts
+
+The followings are the roles needed to deploy API Manager pattern 1.
+
+- apim
+- apim-analytics-worker
+
+### 1. Customize the [inventory](../dev/inventory) file
+
+#### Replace the inventory file content with the followings.
+
+```
+[apim]
+apim_1 ansible_host=[ip_address] ansible_user=[ssh_user]
+apim_2 ansible_host=[ip_address] ansible_user=[ssh_user]
+
+[apim-analytics]
+apim-analytics-worker_1 ansible_host=[ip_address] ansible_user=[ssh_user]
+apim-analytics-worker_2 ansible_host=[ip_address] ansible_user=[ssh_user]
+```
+> NOTE: Replace `[ip_address]` and `[ssh_user]` appropriately.
+
+### 2. Modify the [site.yml](../site.yml) file
+
+```
+---
+# This playbook deploys the whole application stack in this site.
+
+- name: Apply common configuration to all nodes
+  hosts: all
+  roles:
+    - common
+
+- name: Apply API Manager Analytics worker configuration to apim-analytics-worker nodes
+  hosts:
+    - apim-analytics-worker_1
+    - apim-analytics-worker_2
+  roles:
+    - apim-analytics-worker
+
+- name: Apply API Manager configuration to apim nodes
+  hosts:
+    - apim_1
+    - apim_2
+  roles:
+    - apim
+```
+
+### 3. Customize the roles for API Manager pattern 1
+
+```
+.
+└── dev
+    ├── group_vars
+    │   ├── apim-analytics.yml
+    │   └── apim.yml
+    ├── host_vars
+    │   ├── apim_1.yml
+    │   ├── apim_2.yml
+    │   ├── apim-analytics-worker_1.yml
+    │   └── apim-analytics-worker_2.yml
+    └── inventory
+
+```
+API Manager pattern 1 contains 2 groups and the configurations specific for each group should be in the respective yaml file under [group_vars](../dev/group_vars) folder. Configurations specific to each host should be added to the corresponding yaml file under [host_vars](../dev/host_vars) folder.
+
+Most commonly changed values are parameterized in the above files. If further changes are required, the values should be parameterized and added to the files accordingly.
+
+#### i. Customize `apim` role
+
+Navigate to [carbon-home](../roles/apim/templates/carbon-home) of the `apim` role. All the files required to deploy the API Manager active-active combination are here. Follow the instructions in the following document to modify the files.
+- [Configuring an active-active deployment](https://docs.wso2.com/display/AM260/Configuring+an+Active-Active+Deployment)
+
+#### ii. Customize `apim-analytics-worker` role
+
+Navigate to [carbon-home](../roles/apim-analytics-worker/templates/carbon-home) of the `apim-analytics-worker` role. All the files required to deploy the API Manager analytics are here. Follow the instructions in the following files to modify the files.
+- [Configure Analytics](https://docs.wso2.com/display/AM260/Configuring+APIM+Analytics#standardsetup)
+- [Minimum HA deployment](https://docs.wso2.com/display/SP430/Minimum%20High%20Availability%20Deployment)
```

Comment:
@rasanjaya85 should refer to [this](https://apim.docs.wso2.com/en/latest/install-and-setup/setup/distributed-deployment/configure-apim-analytics/active-passive/).